### ISSUE TITLE
4.5.5 ExportSql.class.php incorrect call to SqlParser\Context

### ISSUE THREAD

**Issue by kampalex** [phpmyadmin/phpmyadmin#12029]
Compared to 4.5.4 a Context option is implemented incomplete causing script to fail when $sql_backquotes is not 'checked' in export table view.

Locally fixed by prepending 'SqlParser\' before Context:: (twice) on line 1557.
`                    Context::$MODE |= Context::NO_ENCLOSING_QUOTES;`
to
`                    SqlParser\Context::$MODE |= SqlParser\Context::NO_ENCLOSING_QUOTES;`

### Steps to reproduce
1. Go to table export view
2. Export method: extend, structure and data
3. Leave sql_backquotes checkbox unchecked.
4. Fatal error rises into export (when having display_errors enabled in PHP).

### Expected behaviour
SQL export.

### Actual behaviour
Beginning of SQL export containing
Fatal error: Class 'Context' not found in [...]\libraries\plugins\export\ExportSql.class.php on line 1557

Windows 10 Pro / Apache/2.4.12 / PHP 5.6.16 / MariaDB 10.1.9
phpMyAdmin version: 4.5.5

**Comment by kampalex** [comment]
Seems like the failing code in PMA 4.5.4 was meant for PMA 4.6.0. It was released too early.

In current version PMA 4.6.0, 'SqlParser\Context::' should be reverted back to 'Context::'.
\libraries\plugins\export\ExportSql.php line 1559

To test:
Export (selection of) table(s) and leave 'sql_backquotes' unchecked.

**Comment by ibennetch** [comment]
Sorry I'm not quite following, are you saying you still have a problem with version 4.6.0?

**Comment by kampalex** [comment]
In short: yes.

While using PMA 4.5.5 I made a bug report because an option (sql_backquotes unchecked) didn't work properly. Compared to release 4.5.4 the affected line of code was changed in repository and released in 4.5.5, but that change was meant to be for release 4.6.0. That line of code works properly in 4.6.0, so it was released too early.

The fix I've suggested works in PMA 4.5.5, but it was (unlucky) included in release 4.6.0. Because the way class calls have changed now it causes a regression.
